src/librustc_driver/driver.rs

```diff
@@ -378,9 +380,11 @@ impl<'a, 'b, 'ast, 'tcx> CompileState<'a, 'b, 'ast, 'tcx> {
                          krate: ast::Crate,
                          cstore: &'a CStore)
                          -> CompileState<'a, 'b, 'ast, 'tcx> {
+        let cloned_span = krate.span.clone();
         CompileState {
             krate: Some(krate),
             cstore: Some(cstore),
+            registry: Some(Registry::new(&session, cloned_span)),
```

Comment:
error[E0382]: use of moved value: `krate.span`
   --> src/librustc_driver/driver.rs:386:52
    |
384 |             krate: Some(krate),
    |                         ----- value moved here
385 |             cstore: Some(cstore),
386 |             registry: Some(Registry::new(&session, krate.span)),
    |                                                    ^^^^^^^^^^ value used here after move
    |
    = note: move occurs because `krate` has type `syntax::ast::Crate`, which does not implement the `Copy` trait

error: aborting due to previous error

This was the reason why I made a temporal copy of it.